sdk/test-utils/perf/test/event/mockEventHubConsumerClient.ts

```diff
@@ -37,21 +41,25 @@ export interface EventHandlers {
   processError(err: Error): Promise<void>;
 }
 
-export class MockEventReceiver2 {
+export class MockEventHubConsumerClient {
   private closeCalled: boolean;
   private partitions: number;
   private minDelay: number; // min delay between events in milliseconds
   private maxDelay: number;
   private timers: NodeJS.Timeout[];
   private maxBatchSize: number;
+  private maxEventsPerSecond: number;
+  private maxEventsPerSecondPerPartition: number;
 
   constructor() {
     this.closeCalled = false;
     this.partitions = 12;
-    this.minDelay = 5;
-    this.maxDelay = 10;
+    this.minDelay = 1;
```

Comment:
Why does this class need the concepts of `minDelay` and `maxDelay`?  The .NET version doesn't have these.  The class should raise events as quickly as possible with no delay between them (up to the limit of `maxEventsPerSecond`).

Reply:
Had this from before with the other version of mock receiver, just left it.
Will see again and remove this. 🥲